bindings/nodejs/src/wallet.rs

```diff
@@ -1,50 +1,49 @@
-// Copyright 2021 IOTA Stiftung
+// Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
 use std::sync::Arc;
 
-use iota_sdk::wallet::{
-    events::types::{Event, WalletEventType},
-    message_interface::{
-        create_message_handler, init_logger as init_logger_rust, ManagerOptions, Message, Response,
-        WalletMessageHandler,
+use iota_sdk_bindings_core::{
+    call_wallet_method as rust_call_wallet_method,
+    iota_sdk::wallet::{
+        events::types::{Event, WalletEventType},
+        Wallet,
     },
-    Result,
+    Response, Result, WalletMethod, WalletOptions,
 };
 use neon::prelude::*;
 use tokio::sync::RwLock;
 
-// Wrapper so we can destroy the MessageHandler
-pub type MessageHandlerWrapperInner = Arc<RwLock<Option<MessageHandler>>>;
-// Wrapper because we can't impl Finalize on MessageHandlerWrapperInner
-pub struct MessageHandlerWrapper(pub MessageHandlerWrapperInner);
-impl Finalize for MessageHandlerWrapper {}
+use crate::client::ClientMethodHandler;
 
-pub struct MessageHandler {
+// Wrapper so we can destroy the WalletMethodHandler
+pub type WalletMethodHandlerWrapperInner = Arc<RwLock<Option<WalletMethodHandler>>>;
```

Comment:
good for parties ... say this 10 times in a row without stumbling otherwise you have to drink.

(fun comment ... can be resolved immediately)